### DISCUSSION TITLE
Open Shell "Classic Menu" shows the OS version as Windows 10 on the start menu when installed on Windows 11

### DISCUSSION THREAD

**Discussion by abcxyz0123** [Open-Shell/Open-Shell-Menu#1063] [General]
I found a bug in Open Shell when using the "Classic Menu" with Windows 11 where it will show OS version as Windows 10 on the start menu.  This a a minor annoyance and doesn't at all affect the operation of Open Shell, but I do hope this will get fixed in the next release.

![image](https://user-images.githubusercontent.com/107862236/174620716-ed7fb4eb-2f04-422b-a0f7-c484fe49c161.png)

**Comment by abcxyz0123** [comment]
I want to add to this that I am also using ExplorerPatcher in conjunction with OpenShell on Windows 11 to get the function back for the taskbar that Microsoft nerfed with Windows 11.  Maybe that is what is causing it to read Window 10.  Perhaps OpenShell could be code to use the registry to determine the Windows version if that is not what it does already.  Other then than that OpenShell runs perfect and paired along with ExplorerPatcher everything is exactly what I was accustomed to in Window 8 - 10.

**Comment by tallman69** [comment]
I am also using Open-Shell along with ExplorerPactcher on Windows 11 22H2.  I use the Windows 7 style start menu, so I have never noticed what you are describing.  When I switched to the Classic menu style it also displays Windows 10 Home along the left side.  I have no answer for this behavior.

**Comment by abcxyz0123** [comment]
I just found a workaround to set the OS version name.  In the OpenShell the "Menu Caption" setting is where the display value for the OS version is set on the "Classic Style" menu is set and can be edited.  To make the edit go to the OpenShell settings > Menu Look tab and the Menu Caption can be edited to display.  Easy enough work around to correct the default value that OpenShell is assigning it when installed in Windows 11.

![image](https://user-images.githubusercontent.com/107862236/198860648-219f0a05-160d-4e6a-92b6-3b8553f84d52.png)
![image](https://user-images.githubusercontent.com/107862236/198860654-41137873-ddba-4e40-8963-0883e8b36905.png)
![image](https://user-images.githubusercontent.com/107862236/198860564-9cf32c9f-e59c-4aaf-bb20-bcb96655da8f.png)

**Reply by tallman69** [reply]
Good work!

**Reply by blackcrack** [reply]
added in the Almanach Wiki transscript..
[Classic+Open-Shell_Almanach.odt](https://github.com/Open-Shell/Open-Shell-Menu/files/9895283/Classic%2BOpen-Shell_Almanach.odt)

**Comment by ge0rdi** [comment]
Hello.
Default menu caption was fixed in [4.4.181](https://github.com/Open-Shell/Open-Shell-Menu/releases/tag/v4.4.181).
It should now display proper Windows version out of the box.

**Comment by abcxyz0123** [comment]
Awesome.  Really great to see that this change made it a new build.  Been using this start menu since it was Windows 8 released when it was still known as Classic Shell,  The Window 95 menu was always my favorite Open-Shell lets me still have this (did add the searchbox to that menu it since the is useful).  Then the Quick Launch toolbar is something I have used since Windows 98 (I don't line pinning as the eats up taskbar space) and Microsoft in their stupidity took that away and well as a ton of other taskbar features and even removed the seconds from the clock.  ExplorerPathcher gave that back to me and made the Open-Shell button be able to override the Windows Start Menu button.  Open-Shell and ExplorerPatcher are my go to dynamic duo for making Windows look and work the way I want it to!
